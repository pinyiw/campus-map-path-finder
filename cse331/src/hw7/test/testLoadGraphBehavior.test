# load graph
LoadGraph g heroes.tsv

# print out the nodes
ListNodes g

# print out the children of all nodes
ListChildren g Java
ListChildren g C
ListChildren g Logic
ListChildren g Data
ListChildren g Math
ListChildren g Calculus