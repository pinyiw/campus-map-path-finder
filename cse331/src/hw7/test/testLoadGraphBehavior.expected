# load graph
loaded graph g

# print out the nodes
g contains: C Calculus Data Java Logic Math

# print out the children of all nodes
the children of Java in g are: Data(0.500) Java(0.000) Logic(0.250)
the children of C in g are: C(0.000)
the children of Logic in g are: Data(0.500) Java(0.250) Logic(0.000) Math(1.000)
the children of Data in g are: Data(0.000) Java(0.500) Logic(0.500)
the children of Math in g are: Calculus(1.000) Logic(1.000) Math(0.000)
the children of Calculus in g are: Calculus(0.000) Math(1.000)